Kitodo/src/main/java/org/goobi/production/plugin/CataloguePlugin/QueryBuilder.java

```diff
@@ -59,7 +59,7 @@
  * term “beagle” in the field “4”). Search fields are referenced by the integers
  * used to reference them in PICA library catalogues (i.e. “4” = title, “7” =
  * ISBN, “8” = ISSN, “12” = Record identifier, …; for a list of supported fields
- * see /Goobi/newpages/NewProcess/inc_process.jsp, lines 61 et sqq.)
+ * see /Goobi/pages/NewProcess/inc_process.jsp, lines 61 et sqq.)
```

Comment:
Please adjust the path to the real location. Yes, I know that this kind of change was done by your IDE ;)